lib/ui/painting/canvas.cc

```diff
@@ -86,7 +86,7 @@ fml::RefPtr<Canvas> Canvas::Create(PictureRecorder* recorder,
   return canvas;
 }
 
-Canvas::Canvas(SkCanvas* canvas) : canvas_(canvas) {}
+Canvas::Canvas(SkCanvas* canvas) : canvas_(canvas), image_allocation_size_(0) {}
```

Comment:
fyi some people prefer initialization like this in the header since it's less likely to cause problems in the future if someone adds a constructor.  I'm fine as long as there is an error if it isn't initialized.

Reply:
Done